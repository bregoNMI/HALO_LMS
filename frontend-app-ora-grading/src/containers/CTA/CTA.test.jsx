import React from 'react';
import { shallow } from '@edx/react-unit-test-utils';

import { CTA } from '.';

describe('CTA component', () => {
  test('snapshots', () => {
    const el = shallow(<CTA hide />);
    expect(el.snapshot).toMatchSnapshot();
  });
});
