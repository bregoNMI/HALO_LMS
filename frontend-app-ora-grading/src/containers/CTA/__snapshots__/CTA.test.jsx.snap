// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CTA component snapshots 1`] = `
<PageBanner>
  <span>
    <FormattedMessage
      defaultMessage="Thanks for using the new ORA staff grading experience. "
      description="Thank user for using ora and ask for feed back"
      id="ora-grading.CTA.feedbackMessage"
    />
    <Hyperlink
      destination="https://docs.google.com/forms/d/1Hu1rgJcCHl5_EtDb5Up3hiZ40sSUtkZQfRHJ3fWOvfQ/edit"
      isInline={true}
      showLaunchIcon={false}
      target="_blank"
      variant="muted"
    >
      <FormattedMessage
        defaultMessage="Provide some feedback"
        description="placeholder for the feedback anchor link"
        id="ora-grading.CTA.linkMessage"
      />
    </Hyperlink>
    <FormattedMessage
      defaultMessage=" and let us know what you think!"
      description="inform user to provide feedback"
      id="ora-grading.CTA.letUsKnowMessage"
    />
  </span>
</PageBanner>
`;
